src/components/chart_background.tsx

```diff
@@ -36,7 +36,7 @@ export class ChartBackgroundComponent extends React.Component<ChartBackgroundPro
   render() {
     const { backgroundStyles } = this.props;
 
-    return <div className="echChartBackground" style={{ backgroundColor: backgroundStyles!.color }} />;
+    return <div className="echChartBackground" style={{ background: backgroundStyles!.color }} />;
```

Comment:
I would say avoid `!` on objects cuz if `backgroundStyles` is undefined, this will error.

```suggestion
    return <div className="echChartBackground" style={{ background: backgroundStyles?.color }} />;
```

if it still complains, you could pass a default `backgroundStyles?.color ?? 'blue'`